### PR TITLE
Profile logout

### PR DESCRIPTION
**Pull request by Rockyraven**
added logout page

### CODE REVIEW

File: src/context/videoLikeContext.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+const LikeProvider = ({children}) => {
+
+  const { user } = useVideo();
+  useEffect(()=>{
+    removeLike()
+  },[])
+
+  const removeLike = async (videoID) => {
+    try {
+      console.log(videoID)
```
Comment (irajbhinde) [inline src/context/videoLikeContext.js:17]: You can remove the console logs

File: src/component/remove card/RemoveCard.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+          <div className="video-title">
+            <p className="description">
+             {title}
+            </p>
+            <p className="creater">{creator}</p>
+            <p className="view">{views} view</p>
+          </div>
+        </div>
+      </Link>
+            <p onClick={()=>removeLike(_id)}> <i className="fa fa-trash"></i></p>
```
Comment (irajbhinde) [inline src/component/remove card/RemoveCard.js:31]: Can you check if the remove like functionality is working? I went to liked videos & clicked on trash icon but nothing changed

Also, you could use a better color for icon & add a cursor pointer

**Review by irajbhinde — COMMENTED** [review]
1) Add a cursor pointer on your login / logout page buttons & also on fonts used
2) Add the _redirects to your project as its showing page not found on page refresh

File: src/component/index.js [hunk 3]
```diff
@@ -2,4 +2,5 @@ export { Navbar } from "./navbar/Navbar";
 export { Sidebar } from "./sidebar/Sidebar";
 export { VideoCard } from "./videocard/VideoCard";
 export { Player } from "./player/Player";
-export { SuggestionCard } from "./suggestioncard/SuggestionCard";
\ No newline at end of file
+export { SuggestionCard } from "./suggestioncard/SuggestionCard";
```
Comment (pariyar07) [inline src/component/index.js:5]: Great component namings.

File: src/component/remove card/RemoveCard.js (hunk 2 above)
Comment (pariyar07) [inline src/component/remove card/RemoveCard.js:31]: adding a cursor: pointer is good for UX

File: src/context/videoLikeContext.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+import React, { createContext, useContext, useEffect, useState } from "react";
+import { useVideo } from "./videoContext";
```
Comment (pariyar07) [inline src/context/videoLikeContext.js:3]: Learn about absolute imports if time permits. Makes code cleaner.

File: src/pages/like-page/LikedPage.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+     <div className="video-listing-container">
+        <Sidebar />
+        
+
+          <div className="videocard-list">
+            
+    {likedVideo.map((videos) => (
+      
+      
+      <RemoveCard
```
Comment (pariyar07) [inline src/pages/like-page/LikedPage.js:37]: Please format the code. There are lot of spaces left in the middle.

**Review by pariyar07 — COMMENTED** [review]
I left a few suggestions and check some of the colour contrast on the page like the category button on the Homepage. The other looks good to me.

File: src/context/videoLikeContext.js (hunk 1 above)
Comment (Rockyraven) [reply src/context/videoLikeContext.js:17]: ok I will do it

File: src/component/remove card/RemoveCard.js (hunk 2 above)
Comment (Rockyraven) [reply src/component/remove card/RemoveCard.js:31]: currently, It is not working but I will fix it later

Comment (Rockyraven) [reply src/component/remove card/RemoveCard.js:31]: ok i will do that

**Pull request merged**